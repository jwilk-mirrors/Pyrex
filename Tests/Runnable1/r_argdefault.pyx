def swallow(name = None, airspeed = None, coconuts = None):
	if name is not None:
		print "This swallow is called", name
	if airspeed is not None:
		print "This swallow is flying at", airspeed, "furlongs per fortnight"
	if coconuts is not None:
		print "This swallow is carrying", coconuts, "coconuts"

