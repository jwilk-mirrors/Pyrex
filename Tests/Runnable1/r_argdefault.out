First swallow:
This swallow is called Brian
Second swallow:
This swallow is flying at 42 furlongs per fortnight
Third swallow:
This swallow is carrying 3 coconuts
